app/assets/javascripts/application.js

```diff
@@ -400,3 +400,27 @@ function setPowerState(item, status){
   power_actions.hide();
   $('[rel="twipsy"]').tooltip();
 }
+
+function disableButtonToggle(item) {
+  var isActive = item.classList.contains("active");
+  var formControl = $(item.parentElement.parentElement).children('.form-control');
```

Comment:
`item.closest('.input-group').find('.form-control')` could be used to improve the fragility of relying on the second parent element.